src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorVerifierInProcess.cs

```diff
@@ -244,6 +245,36 @@ public async Task ErrorTagsAsync(string[] expectedTags, CancellationToken cancel
                 string.Join(Environment.NewLine, actualTags));
         }
 
+        public async Task VerifyActiveViewIsInMetadataWorkspaceAsync(CancellationToken cancellationToken)
+        {
+            var activeView = await TestServices.Editor.GetActiveTextViewAsync(cancellationToken);
+            var currentSnapshot = activeView.TextBuffer.CurrentSnapshot;
+            var document = currentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
+            if (document is not null)
+            {
+                Assert.True(document.Project.Solution.Workspace.Kind is WorkspaceKind.MetadataAsSource);
```

Comment:
Assert.Equal here would give a better failure message

Reply:
Rewrote to use `Assert.Equal`